packages/input-option/src/InputOptionContent/InputOptionContent.styles.ts

```diff
@@ -1,42 +1,112 @@
 import { css } from '@leafygreen-ui/emotion';
-import { createUniqueClassName } from '@leafygreen-ui/lib';
-import { spacing } from '@leafygreen-ui/tokens';
+import { createUniqueClassName, Theme } from '@leafygreen-ui/lib';
+import {
+  color,
+  spacing,
+  State,
+  transitionDuration,
+  Variant,
+} from '@leafygreen-ui/tokens';
 
+export const inputOptionContentClassName = createUniqueClassName(
+  'input_option-content',
```

Comment:
Thoughts on adding `input_option` to a constants file?

Reply:
Was planning on tackling that with test harnesses 👍